katsdpingest/utils.py

```diff
@@ -29,9 +29,9 @@ def cbf_telstate_view(telstate: katsdptelstate.TelescopeState,
     # Generate a list of places to look for attributes:
     # - the stream itself
     # - the upstream antenna-channelised-voltage stream, and its instrument
-    src = telstate.view(stream_name, exclusive=True)['src_streams'][0]
+    src = (await telstate.view(stream_name, exclusive=True)['src_streams'])[0]
```

Comment:
Isn't `telstate.view` synchronous?

Reply:
`view` is synchronous, but I then apply `[]` to the result, which is asynchronous.

Reply:
Ah, missed that! What would happen if you removed the `()` around the `await` part?

Reply:
Then the `[0]` would bind before the await.

Reply:
Right, makes sense now.